src/session/session_compact.c

```diff
@@ -498,6 +498,7 @@ __wt_session_compact(WT_SESSION *wt_session, const char *uri, const char *config
     WT_ERR(__wt_config_gets(session, cfg, "timeout", &cval));
     session->compact->max_time = (uint64_t)cval.val;
     __wt_epoch(session, &session->compact->begin);
+    __wt_epoch(session, &session->compact->last_progress);
```

Comment:
We could save a call to read the clock by saying `compact->last_progress = compact->begin`.